### ISSUE TITLE
Don't show "Daily Task Indicator" while on tutorial island.

### ISSUE THREAD

**Issue by SomeZer0** [runelite/runelite#3092]
You can't possibly have access to these dailies yet if you're on tutorial island. (F2P account BTW)
![Image](https://vgy.me/iRkYUA.png)
![Image2](https://vgy.me/78k4Du.png)

**Comment by Infinitay** [comment]
#1129 just waiting on another PR that fixes the event issue and to delay it

**Comment by deathbeam** [comment]
Duplicate of #2337
